src/main/java/io/cdap/plugin/sfmc/source/MarketingCloudClient.java

```diff
@@ -101,10 +102,11 @@ private ETExpression getExpressionfromString(String filter) throws ETSdkExceptio
    * @param object The SourceObject which tells what data to be fetched from Salesforce Marketing Cloud
    * @return The list of ETApiObject representing the records from requested object
    */
-  public List<? extends ETSoapObject> fetchObjectRecords(SourceObject object) throws ETSdkException {
+  public ETResponse<? extends ETSoapObject>  fetchObjectRecords(SourceObject object,
+                                                         String requestId) throws ETSdkException {
```

Comment:
There are still a couple of indent issues like this. You can get the Intellij settings from here -https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard. Please reformat all the changes in PR using these settings. Thanks!

Reply:
Thanks for the jar. reformat all the changes.